compiler/src/iree/compiler/Codegen/SPIRV/Verifiers.cpp

```diff
@@ -150,6 +150,155 @@ LogicalResult verifySPIRVMatmulPromoteVectorizePassPipeline(
   return success();
 }
 
+LogicalResult verifySPIRVCooperativeMatrixVectorizePassPipeline(
+    Operation *op, IREE::Codegen::LoweringConfigAttr loweringConfig,
+    IREE::Codegen::TranslationInfoAttr translationInfo,
+    ArrayRef<int64_t> workgroupSize) {
+  // Verify that the translation info is using the right pipeline
+  if (translationInfo.getDispatchLoweringPassPipeline() !=
+      IREE::Codegen::DispatchLoweringPassPipeline::
+          SPIRVCooperativeMatrixVectorize) {
+    return op->emitOpError("expected pipeline in translation_info to be ")
+           << stringifyEnum(IREE::Codegen::DispatchLoweringPassPipeline::
+                                SPIRVCooperativeMatrixVectorize);
+  }
+
+  if (!isa<linalg::MatmulOp, linalg::BatchMatmulOp>(op)) {
+    return success();
+  }
+  LLVM_DEBUG({
+    llvm::dbgs() << "verifying op: " << *op << "\n";
+    llvm::dbgs() << "chosen workgroup size: [";
+    llvm::interleaveComma(workgroupSize, llvm::dbgs());
+    llvm::dbgs() << "]\n";
+  });
+
+  // Get spirv.target_env attributes
```

Comment:
Nit: place `.` to end the sense for the comment. Also for the following comments.